### PR TITLE
feat(tutorial): update wrapping up step in the tutorial

### PR DESCRIPTION
**Pull request by jeffchew**
### Related Ticket(s)

Refs https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7175

### Description

This includes minor updates to the `wrap-up` step of the `Carbon for IBM.com Web Components` tutorial. Since it will be also going to the badge page for submitting, the text is largely the same as the Carbon tutorial for React.

### Changelog

**Changed**

- Add page titles to the list of steps
- Add badge url to the wrap-up page

### CODE REVIEW

File: src/pages/developing/web-components-tutorial/overview.mdx [hunk 1]
```diff
@@ -76,11 +76,11 @@ any step and take it from there.
 
 1. [**Installing Carbon**](/developing/web-components-tutorial/step-1)
```
Comment (ariellalgilmore) [inline src/pages/developing/web-components-tutorial/overview.mdx:77]: Ignacio just updated this section too in this [pr](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/pull/1396) 
```suggestion
1. [**Installing Carbon for IBM.com**](/developing/web-components-tutorial/step-1)
```

**Review by ariellalgilmore — APPROVED** [review]
One small comment, but everything else looks great!

File: src/pages/developing/web-components-tutorial/wrapping-up.mdx [hunk 2]
```diff
@@ -39,23 +39,23 @@ tutorial, not just IBM employees.
...
 badge:
 
 1. Complete steps 1 through 5 of the Carbon for IBM.com Web Components tutorial:
    - Step 1. [Installing Carbon](/developing/web-components-tutorial/step-1)
-   - Step 2. [TBD](/developing/web-components-tutorial/step-2)
-   - Step 3. [TBD](/developing/web-components-tutorial/step-3)
-   - Step 4. [TBD](/developing/web-components-tutorial/step-4)
-   - Step 5. [TBD](/developing/web-components-tutorial/step-5)
+   - Step 2. [Adding out of the box components](/developing/web-components-tutorial/step-2)
+   - Step 3. [Adding custom Content Block components](/developing/web-components-tutorial/step-3)
```
Comment (janchild) [inline src/pages/developing/web-components-tutorial/wrapping-up.mdx:53]:
```suggestion
   - Step 3. [Adding custom Content block components](/developing/web-components-tutorial/step-3)
```
```suggestion
   - Step 3. [Adding custom Content Block components](/developing/web-components-tutorial/step-3)
```

File: src/pages/developing/web-components-tutorial/wrapping-up.mdx [hunk 3]
```diff
@@ -39,23 +39,23 @@ tutorial, not just IBM employees.
...
+   - Step 4. [Adding custom Carbon components](/developing/web-components-tutorial/step-4)
+   - Step 5. [Deploying to Github Pages](/developing/web-components-tutorial/step-5)
```
Comment (janchild) [inline src/pages/developing/web-components-tutorial/wrapping-up.mdx:55]:
```suggestion
   - Step 5. [Deploying to GitHub Pages](/developing/web-components-tutorial/step-5)
```
```suggestion
   - Step 5. [Deploying to Github Pages](/developing/web-components-tutorial/step-5)
```

**Review by janchild — APPROVED** [review]
Just a couple of little copy tweaks, but approving!

**Pull request merged**